src/status_im2/contexts/chat/composer/actions/view.cljs

```diff
@@ -129,7 +129,11 @@
                                                     (alert/show-popup
                                                      (i18n/label :t/audio-recorder-error)
                                                      (i18n/label
-                                                      :t/audio-recorder-permissions-error)))
+                                                      :t/audio-recorder-permissions-error)
+                                                     nil
+                                                     {:text (i18n/label :t/settings)
+                                                      :accessibility-label :settings-button
+                                                      :onPress (fn [] (permissions/open-settings))}))
```

Comment:
its better to do not use effects directly and dispatch

Reply:
That's a function from the `react-native-permissions` library.

https://github.com/status-im/status-mobile/pull/16049/files#diff-56a9997cbefcb754a52533b053549104a87331bd433463fbc5ee56af7dd56f64R42

Reply:
yes, i know, that's why I'm saying it shouldn't be used directly, but it should be used as an effect in the event handler